modules/nlu/src/backend/api.ts

```diff
@@ -118,6 +118,64 @@ export const registerRouter = async (bp: typeof sdk, app: NLUApplication) => {
       return mapError({ botId, lang, error }, res)
     }
   })
+
+  addTrainingServiceRoutes(router, app, mapError)
+}
+
+const trainingStatusSchema = Joi.object().keys({
+  status: Joi.string()
+    .valid(['idle', 'done', 'needs-training', 'training-pending', 'training', 'canceled', 'errored'])
+    .required(),
+  progress: Joi.number()
+    .default(0)
+    .min(0)
+    .max(1)
+    .required()
+})
+
+const addTrainingServiceRoutes = (
```

Comment:
Can you add a documentation for this API in comments ? Just so we can point out to this file if something goes wrong. Internal backdoor or something.

```ts
/**
* This is the training repo's API blah blah
* Here's the routes:
* POST 'BP_HOST:PORT/api/v1/mod/nlu/...' 
*/
```